openmicroscopy/openmicroscopy
# Delete Tags

This fixes tag deletion and permissions issues with all deletes.

To test:
 - Log in to Read-Write group.
 - Show Tag tree for "All Members" - Check that root of tree shows "All Members"
 - Select a mixture of Tags that you own or don't own.
 - Delete button should be enabled - click it.
 - Check that delete message makes sense.
 - Go ahead with delete and check that everything works as expected.
 - Check that in a group with other permissions that that Delete is not enabled for other users' tags unless you are Admin or Group Owner.
 - Confirm that deletion of other objects (P/D/I/S/P) is still OK, since code has been refactored to avoid duplication.
